utils/http.py

```diff
@@ -12,3 +12,15 @@ def retrieve_json(url, timeout=DEFAULT_TIMEOUT, verify=True):
     r = requests.get(url, timeout=timeout, verify=verify)
     r.raise_for_status()
     return r.json()
+
+# Get expvar stats
+def get_expvar_stats(key, port):
+    try:
+        json = retrieve_json("http://127.0.0.1:{port}/debug/vars".format(port=port))
```

Comment:
ok, so since dogstatsd6 uses `localhost`, can we use `localhost` instead of `127.0.0.1`?

There's some logic in the agent5 (https://github.com/DataDog/dd-agent/blob/5.14.1/config.py#L269-L275) that falls back to `127.0.0.1` only if localhost can't be resolved, so if agent6 always uses `localhost` should we just stick to `localhost` here?

That said I don't think this is that important, and at some point we'll probably want to improve the logic on dogstatsd6's side